frontends/open-discussions/src/components/PageBanner.js

```diff
@@ -0,0 +1,179 @@
+// @flow
+import * as React from "react"
```

Comment:
No changes here besides removing TS definitions. This is generally the same as `ol-util/src/components/PageBanner.ts`.